ci/upgrade/setup.sh

```diff
@@ -28,7 +28,7 @@ sleep 100
 
 echo "Wait for cspc-operator to start"
 
-kubectl wait --for=condition=available --timeout=300s deployment/cspc-operator -n openebs
+kubectl wait --for=condition=available --timeout=550s deployment/cspc-operator -n openebs
```

Comment:
n_: update the PR on why this change was required..

Reply:
Done